pkg/maps/ctmap/ctmap.go

```diff
@@ -294,7 +294,7 @@ func newMap(mapName string, m mapType) *Map {
 }
 
 func purgeCtEntry6(m *Map, key CtKey, natMap NatMap) error {
-	err := m.Delete(key)
+	err := m.DeleteIgnoreMetricErrorCount(key, true)
```

Comment:
You probably want to use the method at the line 299 instead.

Reply:
@brb I think I am confused here. I thought you want me to replace the 'DeleteIgnoreMetricErrorCount' function with 'DeleteMapping' which is at line 299.
Can you clarify here.